jpo-ode-svcs/src/main/resources/templates/index.html

```diff
@@ -17,29 +17,26 @@ <h2 style="color: #ff0000">Seems your browser doesn't support Javascript! Websoc
     enabled. Please enable
     Javascript and reload this page!</h2></noscript>
     <nav class="navbar navbar-inverse navbar-fixed-top">
-      <div class="container">
-          <a class="navbar-brand" href="#">BSM ODE</a>
-        </div>
+        <div class="container"> <a class="navbar-brand" href="#">BSM ODE</a> </div>
```

Comment:
Is this still an appropriate title since we are doing more than BSMs? Let's come up with a more generic title like "ODE Demo Console" or something in those lines.